master/config.sls

```diff
@@ -0,0 +1,24 @@
+{% from "master/map.jinja" import master with context %}
+
+include:
+  - .services
```

Comment:
Is it ok that the minion/master services may or may not start after the minion/extra configs are deployed? AFAIK include doesn't guarantee execution ordering on it's own.